lib/consult/cli.rb

```diff
@@ -26,6 +26,7 @@ def parse_options(argv)
       opts = {
         config_dir: Dir.pwd,
         force_render: true,
+        raise_on_error: false,
```

Comment:
This boolean field should be an enum, eg `:log` or `:raise`

```suggestion
        error: :log
```